include/swift/AST/Types.h

```diff
@@ -3117,6 +3187,21 @@ class FunctionType final : public AnyFunctionType,
     return {getTrailingObjects<Param>(), getNumParams()};
   }
 
+  const clang::Type *getClangFunctionType() const {
+    switch (numTrailingObjects(OverloadToken<ExtInfo::Uncommon>())) {
```

Comment:
?

Reply:
I'm guessing the ? was for "why is this implementation unnecessarily complicated"; fixed that.

Reply:
Thanks!  Looks a lot better.